Geno2MP.pm

```diff
@@ -0,0 +1,210 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ Geno2MP
+
+=head1 SYNOPSIS
+
+ cp Geno2MP.pm ${HOME}/.vep/Plugins
+ ./vep -i variations.vcf --plugin Geno2MP,file=/path/to/Geno2MP/data.vcf.gz
+
+ # Return more columns from Geno2MP VCF file
+ ./vep -i variations.vcf --plugin Geno2MP,file=/path/to/Geno2MP/data.vcf.gz,cols=HPO_CT:FXN:nhomalt_male_aff:nhomalt_male_unaff
+
+ # Build and return Geno2MP URL based on GRCh37 variant location
+ ./vep -i variations.vcf --plugin Geno2MP,file=/path/to/Geno2MP/data.vcf.gz,url=1
+
+=head1 DESCRIPTION
+
+ A VEP plugin that adds information from Geno2MP, a web-accessible database of
+ rare variant genotypes linked to phenotypic information.
+
+ Parameters can be set using a key=value system:
+   file: VCF file containing Geno2MP data
+   cols: colon-delimited list of Geno2MP columns to return from INFO fields
+         (by default it only returns the column HPO_CT)
+   url: build and return URL to Geno2MP variant page (boolean; 0 by default);
```

Comment:
You update the docs here saying by default URL is 0 but in the code the default is still 1.

Reply:
🤦‍♂️ Sorry, it's now fixed!